src/h5web/visualizations/containers/NxImageContainer.tsx

```diff
@@ -16,9 +21,16 @@ function NxImageContainer(props: VisContainerProps): ReactElement {
   assertGroup(entity);
 
   const { metadata } = useContext(ProviderContext);
-  const signal = getAttributeValue(entity, 'signal');
+
+  const NxData = getNxDataGroup(entity, metadata);
+  if (!NxData) {
```

Comment:
These variables (here and in the other container) need to start with a lowercase.